openid-4-verifiable-presentations-1_0.md

```diff
@@ -2447,10 +2448,8 @@ A non-normative example of the Authorization Response would look the same as in
 
 The following requirements apply to the `nonce` and `aud` claims in the Key Binding JWT:
 
-- the `nonce` claim MUST be the value of `nonce` from the Authorization Request;
-- the `aud` claim MUST be the value of the Client Identifier; 
-
-Note that for an unsigned Authorization Request over the DC API, the `client_id` parameter is not used. Instead, the effective Client Identifier is derived from the Origin, as described in (#dc_api_request).
```

Comment:
I would keep below clarification.  it feels important to repeat that client_id is omitted in unsigned dc api requests
> Note that for an unsigned Authorization Request over the DC API, the `client_id` parameter is not used.

Reply:
I was hoping that the addition of 2059 would make that unnecessary. I would like to make it clear that for DC API unsigned, the client_id is constructed by the wallet and not have to repeat that for every credential format